### PR TITLE
Feature/docs sg

### PR DESCRIPTION
**Pull request by ThornWalli**

### CODE REVIEW

File: example/components/organisms/PreviewContainer.vue [hunk 1]
```diff
@@ -110,31 +112,25 @@ export default {};
...
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+      height: 100%;
 
-        & > ul {
-          padding: 0;
-          margin: 0;
-          list-style: none;
+      & >>> ul {
```
Comment (StephanGerbeth) [inline example/components/organisms/PreviewContainer.vue:123]: lets use a class selector

File: docs/content/v2/en/options.md [hunk 2]
```diff
@@ -14,6 +14,23 @@ category: Guide
 Sets the global `crossorigin` value of the nuxt-speedkit preloads.  
 The default value is the `crossorigin` value from the [Render Configuration](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-render#crossorigin).
 
+## `optimizePreloads`
+- Type: `Boolean`
+  - Default: `true`
+
+If set, the preloads and scripts will be optimized.
```
Comment (StephanGerbeth) [inline docs/content/v2/en/options.md:21]: Activating this option will optimize the initial script preloads

File: docs/content/v2/en/options.md [hunk 3]
```diff
@@ -14,6 +14,23 @@ category: Guide
...
+
+The following nuxt options are set:
```
Comment (StephanGerbeth) [inline docs/content/v2/en/options.md:23]: The following NuxtJS settings are made or overwritten in the nuxt.config:

File: docs/content/v2/en/options.md [hunk 4]
```diff
@@ -14,6 +14,23 @@ category: Guide
...
+
+| Property                            | Value   |
+| ----------------------------------- | ------- |
+| `nuxt.options.generate.manifest`    | `false` |
+| `nuxt.options.render.resourceHints` | `true`  |
+| `nuxt.options.render.asyncScripts`  | `true`  |
+| `nuxt.options.render.http2.push`    | `false` |
+
+And not used `client` & `modern` scripts tags filtered out.
```
Comment (StephanGerbeth) [inline docs/content/v2/en/options.md:32]: ????

**Review by StephanGerbeth — CHANGES_REQUESTED** [review]

File: example/components/organisms/PreviewContainer.vue (hunk 1 above)
Comment (ThornWalli) [reply example/components/organisms/PreviewContainer.vue:123]: Angepasst!

File: docs/content/v2/en/options.md (hunk 3 above)
Comment (ThornWalli) [reply docs/content/v2/en/options.md]: Angepasst!

File: lib/components/SpeedkitYoutube.vue [hunk 5]
```diff
@@ -77,22 +88,27 @@ export default {
...
   }
 }
 
->>> {
-  @nest &:hover {
-    & .play path:first-child {
-      fill: #f00;
-      fill-opacity: 1;
-      transition: fill 0.1s cubic-bezier(0, 0, 0.2, 1), fill-opacity 0.1s cubic-bezier(0, 0, 0.2, 1);
+/* .nuxt-speedkit-youtube {
```
Comment (StephanGerbeth) [inline lib/components/SpeedkitYoutube.vue:105]: can we remove that?

**Review by StephanGerbeth — CHANGES_REQUESTED** [review]

**Review by StephanGerbeth — APPROVED** [review]

**Review by StephanGerbeth — APPROVED** [review]

File: lib/components/SpeedkitYoutube.vue (hunk 5 above)
Comment (ThornWalli) [reply lib/components/SpeedkitYoutube.vue]: comment is removed

**Review by StephanGerbeth — APPROVED** [review]

**Pull request merged**